inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -219,7 +219,11 @@ class MKLDNNGraph {
 
 private:
     void EnforceBF16();
-    void printGraphInfo() const;
+#ifdef CPU_DEBUG_CAPS
```

Comment:
Would be much better to isolate those function in scope of some separate file, but since they use few private methods and fields,  it will require addition of multiple **friends** relationships

Reply:
For me it sounds like we should make public APIs for these stuff.

Reply:
better to be discuss

Reply:
Moved to mkldnn_graph_dumper.cpp. Introduced few new public const functions for mkldnn_edge.h, mkldnn_node.h